src/content/data-streams/billing.mdx

```diff
@@ -0,0 +1,26 @@
+---
+section: dataStreams
+title: "Data Streams Billing"
+isIndex: false
+whatsnext:
+  {
+    "Learn the basics about how to retrieve Data Streams reports in the Getting Started guide.": "/data-streams/getting-started",
+    "Find the list of available stream IDs.": "/data-streams/stream-ids",
+    "Find the schema of data to expect from Data Streams reports.": "/data-streams/reference/report-schema",
+  }
+---
+
+import { Aside, ClickToZoom } from "@components"
+
+<Aside type="note" title="Mainnet Access">
+  Chainlink Data Streams is available on Arbitrum Mainnet and Arbitrum Sepolia.
+</Aside>
+
+<Aside type="note" title="Talk to an expert">
+  <a href="https://chainlinkcommunity.typeform.com/datastreams?#ref_id=docs">Contact us</a> to talk to an expert about
+  integrating Chainlink Data Streams with your applications.
+</Aside>
+
+Chainlink Data Streams supports fee payments in LINK and in alternative assets, which currently includes native blockchain gas tokens and their ERC20-wrapped version. Payments made in alternative assets have no surcharge when compared to LINK payments. You pay to verify reports from Data Streams onchain using the verifier contract. The verification price depends on the Data Streams feed ID from which you verify a report. You pay per report verified. If you verify multiple reports in a batch, you pay for all of the reports included in that batch.
```

Comment:
```suggestion
Chainlink Data Streams supports fee payments in LINK and in alternative assets, which currently includes native blockchain gas tokens and their ERC20-wrapped version. There is no additional surcharge for paying fees with alternative assets instead of LINK.

You pay to verify reports from Data Streams onchain using the verifier contract. The verification price depends on the Data Streams feed ID from which you verify a report. You pay per report verified. If you verify multiple reports in a batch, you pay for all of the reports included in that batch.
```